### ISSUE TITLE
Build Failed

### ISSUE THREAD

**Issue by pbeppe** [kfiross/flutter_mongo_stitch#38]
i have this error: 
Flutter_mongo_stitch\FlutterMongoStitchPlugin.kt: (133, 33): Type mismatch: inferred type is String? but String was expected
What went wrong:
Execution failed for task ':flutter_mongodb_realm:compileDebugKotlin'.
> Compilation error. See log for more details

**Comment by kfiross** [comment]
Hi @pbeppe
I will check soon why this happens, which version of the plugin you used?

**Comment by pbeppe** [comment]
I use 1.2.6 with Flutter beta channel

**Comment by kfiross** [comment]
ok, thanks

**Comment by kfiross** [comment]
@pbeppe 
Did you tried to delete cache and build folder?

**Comment by pbeppe** [comment]
yes i tried

**Comment by mosheap** [comment]
Hey, 
Got the same problem... 1.2.6 with flutter stable 2.3
There is any solution?

**Comment by pbeppe** [comment]
Ok thank you a try it


Giuseppe Palmisano


  [Descrizione: Descrizione: Descrizione: logo mail]

Pro Energy Group snc
Di Mastroleo A. & C
Via Giardini Nord 75 - 41026 Pavullo n/F (MO) Italia
Tel +39 0536324837 Fax +39 05361851029 Num.Verde 800364800
C.F. - P.Iva 03176800369

***@***.***


In ottemperanza alla Legge 196/2003 sulla tutela dei dati personali, le informazioni contenute in questo messaggio sono strettamente riservate ed esclusivamente indirizzate al destinatario indicato (oppure alla persona, responsabile di rimetterlo al destinatario), tenendo presente che qualsiasi uso, riproduzione o divulgazione di questo messaggio è vietata.
Nel caso in cui aveste ricevuto questa mail per errore, vogliate avvertire il mittente al più presto e subito dopo distruggerla. Grazie.
Pursuant to Legislative Decree No. 196/2003, you are hereby informed that this message contains confidential information intended only for the use of the addressee. If you are not the addressee, and have received this message by mistake, please delete it and immediately notify us. You may not copy or disseminate this message to anyone. Thank you.

Da: mosheap ***@***.***>
Inviato: mercoledì 11 agosto 2021 08:05
A: kfiross/flutter_mongodb_realm ***@***.***>
Cc: pbeppe ***@***.***>; Mention ***@***.***>
Oggetto: Re: [kfiross/flutter_mongodb_realm] Build Failed (#38)


Hey,
Got the same problem... 1.2.6 with flutter stable 2.3
There is any solution?

-
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fkfiross%2Fflutter_mongodb_realm%2Fissues%2F38%23issuecomment-896527550&data=04%7C01%7C%7C2c69e7957e1f464b7f0008d95c8df4fe%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637642587032917269%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=2wZkApkVXRrwdjQ7Cy2OFJEsNv98ibMOP2%2BzKgN4Iu4%3D&reserved=0>, or unsubscribe<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAC452UZ3A2QGGQZ4LIGH36DT4IHI3ANCNFSM46HXCLLA&data=04%7C01%7C%7C2c69e7957e1f464b7f0008d95c8df4fe%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637642587032917269%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=W80J3Nmqx7zsdmmzXDtxiMmUaSuL4%2BllgDUn7lI3nZM%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7C%7C2c69e7957e1f464b7f0008d95c8df4fe%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637642587032927225%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=yPP6IwybIORKRd2FI8AnsW6ITFaevJhQ64D8iAJbShc%3D&reserved=0> or Android<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26utm_campaign%3Dnotification-email&data=04%7C01%7C%7C2c69e7957e1f464b7f0008d95c8df4fe%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637642587032927225%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=QGoPn8WOB%2FK0DfILAn9WXhuHRaiuXAOBX1uIbjdaxg0%3D&reserved=0>.

**Comment by richy7734** [comment]
Did anyone get the solution?

**Comment by pbeppe** [comment]
No. I use the last version of flutter and don't work with plug-in.


Giuseppe Palmisano


  [Descrizione: Descrizione: Descrizione: logo mail]

Pro Energy Group snc
Di Mastroleo A. & C
Via Giardini Nord 75 - 41026 Pavullo n/F (MO) Italia
Tel +39 0536324837 Fax +39 05361851029 Num.Verde 800364800
C.F. - P.Iva 03176800369

***@***.***


In ottemperanza alla Legge 196/2003 sulla tutela dei dati personali, le informazioni contenute in questo messaggio sono strettamente riservate ed esclusivamente indirizzate al destinatario indicato (oppure alla persona, responsabile di rimetterlo al destinatario), tenendo presente che qualsiasi uso, riproduzione o divulgazione di questo messaggio è vietata.
Nel caso in cui aveste ricevuto questa mail per errore, vogliate avvertire il mittente al più presto e subito dopo distruggerla. Grazie.
Pursuant to Legislative Decree No. 196/2003, you are hereby informed that this message contains confidential information intended only for the use of the addressee. If you are not the addressee, and have received this message by mistake, please delete it and immediately notify us. You may not copy or disseminate this message to anyone. Thank you.

Da: richy7734 ***@***.***>
Inviato: domenica 5 settembre 2021 11:34
A: kfiross/flutter_mongodb_realm ***@***.***>
Cc: pbeppe ***@***.***>; Mention ***@***.***>
Oggetto: Re: [kfiross/flutter_mongodb_realm] Build Failed (#38)


Did anyone get the solution?

-
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fkfiross%2Fflutter_mongodb_realm%2Fissues%2F38%23issuecomment-913117818&data=04%7C01%7C%7Cedb0f26cd5374157525908d970504c2b%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637664312435757235%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=MHvneShMZxJR3mhBvoGdK0U4UIf3VgngED6qNGT9h3s%3D&reserved=0>, or unsubscribe<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAC452UZT7IXAXW5JQJNLAL3UAM2QTANCNFSM46HXCLLA&data=04%7C01%7C%7Cedb0f26cd5374157525908d970504c2b%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637664312435757235%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=b53cG2lp7jmgFYhUbBtKP%2Fzf9vjxgufY6KPbBu55hAo%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7C%7Cedb0f26cd5374157525908d970504c2b%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637664312435767192%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=HkGiE8do9PMvGay%2B1agpWzfhZ%2Bsd3Ud0LMEt8Vo7Lxk%3D&reserved=0> or Android<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7C%7Cedb0f26cd5374157525908d970504c2b%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637664312435767192%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=6IwpJZl78a3cTiZ7PL5BWMd2iiG5LNJkPJjKxyYcVz8%3D&reserved=0>.

**Comment by mwillner** [comment]
Hey @kfiross,

sorry to bother you, but do you have any news on this one?
Or could anyone else fix this? 

Any hint would be much appreciated.
Thx

**Comment by LeonManolo** [comment]
Hey,
Im unfortunately also getting this error while using the newest null safe version.

```
e: /Users/manolo/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_mongodb_realm-2.0.0-nullsafety.2/android/src/main/kotlin/com/example/flutter_mongo_stitch/FlutterMongoStitchPlugin.kt: (133, 33): Type mismatch: inferred type is String? but String was expected

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':flutter_mongodb_realm:compileDebugKotlin'.
> Compilation error. See log for more details
```

**Comment by kfiross** [comment]
@LeonManolo  fixed on version 2.0.0
